interface/forms/eye_mag/help.php

```diff
@@ -35,411 +34,572 @@
-                        <textarea class="kb_entry">D;bll:+2 meibomitis;rll:frank ect, 7x6mm lid margin bcc lat.a;bul:2mm ptosis;rul.+3 dermato.a</textarea>
-                        <button id="external_button2">Details</button>
-                        <br /><h4 class="bold">Anterior Segment:</h4>
-                        <textarea class="kb_entry">D;bc:+2 inj;bk:med pter;rk:mod endo gut.a;bac:+1 fc, +1 pig cells</textarea>
-                        <button id="antseg_button2">Details</button>
+                            <h4 class="font-weight-bold mt-5">PMH: </h4>
+                            <textarea class="form-control">POH: POAG.Myopia. Dry Eye; POS:Phaco/IOL OD 4/4/1994.Phaco/IOL OS 4/24/1995. Yag/PCO OD 6/5/1999;Meds:Timolol 0.5% GFS QHS OU. Latanoprost 0.01% QHS OU. Trazadone 50mg PO QHS.Famvir 500mg PO TID;Surg:Appendectomy 1998. Choly 2010.Lap Band 2014.;All:sulfa - hives.PCN - SOB;</textarea>
+                            <button class="btn btn-primary mt-2" id="pmh_button2">Details</button>
+                            <br />
 
-                        <br />
-                        <h4 class="bold">Retina:</h4>
-                        <textarea class="kb_entry">D;bd:+2 bowtie pallor;rcup:0.6Vx0.4H w/ inf notch;lcup:0.5;rmac:+2 BDR, +CSME;lmac:flat, tr BDR;v:+PPDR, ++venous beading;rp:ht 1 o,no vh;</textarea>
-                        <button id="retina_button2">Details</button>
-
-                        <h4 class="bold">Strabismus:</h4>
-                        <textarea class="kb_entry">scDist;5:8ix 1rht;4:10ix;6:6ix;2:15xt;8:5ix;ccDist;4:5ix;5:ortho;6:ortho</textarea>
-                        <button id="neuro_button2">Details</button>
-
-                        <hr />
-                        Below all these lines are strung together. Copy and paste this into a test patient's chart.  <br />
-                        Voila! HPI, PMH, POH, Medications Allergies and 40 clinical findings + normals, are documented.
-                        <hr />
-
-                        <textarea class="kb_entry" style="height:2.3in;">CC:"My eyes are tearing and there is a yellow discharge";hpi:The symptoms began last week and the discharged turned yellow yesterday.  No photophobia.  The redness spread from the right to the left eye two days ago.;
-POH:POAG. Myopia. Dry Eye; POS: Phaco/IOL OD 4/4/1994.Phaco/IOL OS 4/24/1995. Yag/PCO OD 6/5/1999;Meds:Timolol 0.5% GFS QHS OU. Latanoprost 0.01% QHS OU.
-Trazadone 50mg PO QHS.Famvir 500mg PO TID;Surg:Appendectomy 1998. Choly 2010.Lap Band 2014.;All:sulfa - hives.PCN - SOB;
-D;bll:+2 meibomitis;rll:frank ect, 7x6mm lid margin bcc lat.a;bul:2mm ptosis;rul:+3 dermato.a
-bc:+2 inj;bk:med pter;rk:mod endo gut.a;bac:+1 fc, +1 pig cells;
-bd:+2 bowtie pallor;rcup:0.6Vx0.4H w/ inf notch;lcup:0.5;rmac:+2 BDR, +CSME;lmac:flat, tr BDR;v:+PPDR, ++venous beading;rp:ht 1 o,no vh;
-scDist;5:8ix 1rht;4:10ix;6:6ix;2:15xt;8:5ix;ccDist;4:5ix;5:ortho;6:ortho</textarea>
```

Comment:
Hello @bradymiller  and @tywrenn 

Can I change the textarea content indent?

Reply:
@stu01509 yeah you can just make sure it passes php codesniffer tests and that the format of the text doesn't change